src/vs/workbench/browser/workbench.ts

```diff
@@ -328,6 +328,7 @@ export class Workbench extends Layout {
 		// Create Parts
 		[
 			{ id: Parts.TITLEBAR_PART, role: 'contentinfo', classes: ['titlebar'] },
+			{ id: Parts.BANNER_PART, role: 'none', classes: ['banner'] },
```

Comment:
maybe role should be [alert](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Alert_Role) or [status](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_status_role)

Reply:
Alert seems more suitable.